protoc-gen-docs/htmlGenerator.go

```diff
@@ -366,6 +365,9 @@ func (g *htmlGenerator) generateFileHeader(top *fileDescriptor, numEntries int)
 		if top == nil {
 			croak("PANIC: null file %v", name)
 		}
+		if top.topMatter == nil {
```

Comment:
Could this behave like the other warnings that this tool prints? For example, the tool outputs warnings if a field has no comments. Seems like it could just print a warning about missing file description and continue processing, no?